src/logger/luaLoggerBinding.c

```diff
@@ -550,12 +553,12 @@ int Lua_AddVirtualChannel(lua_State *L){
 	size_t args = lua_gettop(L);
 	if (args >= 2){
 		ChannelConfig chCfg;
-		strncpy(chCfg.label, lua_tostring(L, 1), DEFAULT_LABEL_LENGTH);
+		strncpy(chCfg.label, lua_tostring(L, 1), DEFAULT_LABEL_LENGTH - 1);
```

Comment:
No.  This will only copy up to 11 characters and possibly not the null terminated character.  Need to use strnlcpy.  See my patch: https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/41